platform-api/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/dao/UserDao.java

```diff
@@ -73,10 +77,21 @@
      * Gets user from persistent layer by his identifier
      *
      * @param id
-     *         user name or identifier
+     *         user identifier
      * @return user POJO
-     * @throws org.eclipse.che.api.core.NotFoundException
+     * @throws NotFoundException
      *         when user doesn't exist
      */
     User getById(String id) throws NotFoundException, ServerException;
+
+    /**
+     * Gets user from persistent layer by his username
+     *
+     * @param userName
```

Comment:
I think It would be better to rename this parameter to user if it can equals to name or id.